### PR TITLE
Static analysis introductory blog post

### PR DESCRIPTION
**Pull request by alice-i-cecile**

### CODE REVIEW

File: content/post/static-analysis-for-games.md [hunk 1]
```diff
@@ -1,12 +1,101 @@
...
+As a result, classical playtesting should often be considered as either a last resort, or as a "finishing pass" on a design you've already analyzed on it's own.
+This is particularly easy and effective if you have good tuning levers to work with, allowing your classical playtest to focus on nailing those values and improving game feel.
+
+## Conclusion
+
+In the end, classical playtesting can still be fun, informative, and highly effective, particularly for learning about subjective player experience, and performing holistic evaluation.
+But, as we hope we've convinced you in this blog post, it's only one tool in your toolbox.
+Use static analysis first to identify problems, hammer them out with focused tests and then use classical playtesting to refine feel and catch unforeseen issues.
+By doing this, you can catch mistakes early and guide your designs much more intentionally.
+As you sharpen these analysis skills, you'll eventually be able to accept, reject, and critique whole systems by inspection alone.
```
Comment (alice-i-cecile) [inline content/post/static-analysis-for-games.md:101]: Rereading, I really dislike this line. It's not a strong ending, and doesn't add anything interesting.

Comment (plof27) [reply content/post/static-analysis-for-games.md:101]: I still really like it, but I'm more than willing to bend. Suggestions for alternates?

Comment (alice-i-cecile) [reply content/post/static-analysis-for-games.md:101]: Okay so the *actual* problem here is that "Conclusion" is, in general, a bad section. It's purposeless and just serves to rehash stuff people read just prior.

Instead, we need to be writing with a goal. In this case, I'm going to go with "Every tool has a place"

**Comment by alice-i-cecile** [comment]
I think this is ready for final review :D One out of two initial blog posts complete!

File: content/post/static-analysis-for-games.md [hunk 2]
```diff
@@ -92,10 +92,15 @@ Testing your game from end-to-end, as if you were a player, can be great for cat
...
-## Conclusion
+## Every tool has a place
 
 In the end, classical playtesting can still be fun, informative, and highly effective, particularly for learning about subjective player experience, and performing holistic evaluation.
 But, as we hope we've convinced you in this blog post, it's only one tool in your toolbox.
 Use static analysis first to identify problems, hammer them out with focused tests and then use classical playtesting to refine feel and catch unforeseen issues.
 By doing this, you can catch mistakes early and guide your designs much more intentionally.
-As you sharpen these analysis skills, you'll eventually be able to accept, reject, and critique whole systems by inspection alone.
+
+As we've built games together, we've learned to **turn patterns into tools.**
```
Comment (plof27) [inline content/post/static-analysis-for-games.md:102]: This new section is nice, but I don't feel like it fits. The conclusion isn't the place to be bringing in new information in my opinion. It should be a short summary of the takeaways, and maybe a call to action. I'm in favor of just cutting this and putting it in a different post, but I'm fine with just moving it to a different section. If you want to change the original ending line, you still have my blessing.

**Review by plof27 — CHANGES_REQUESTED** [review]

File: content/post/static-analysis-for-games.md (hunk 2 above)
Comment (alice-i-cecile) [reply content/post/static-analysis-for-games.md:102]: Definitely agree on the call to action. Hmm...

**Comment by alice-i-cecile** [comment]
Done! Turn off draft mode when we're ready to publish.

**Pull request merged**